docs/source/developers/architecture.rst

```diff
@@ -71,3 +71,95 @@ Jupyter Server contains the following components:
   - **Kernel Spec Manager** parses files with JSON specification for a Kernels,
     and provides a list of available Kernel configurations. To learn about
     Kernel Spec, check `the Jupyter Client guide <https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernel-specs>`_.
+
+Create Session Workflow
+-----------------------
+
+The create Session workflow can be seen in figure below:
+
+.. image:: ../images/session-create.drawio.png
+   :alt: Create Session Workflow
+   :width: 90%
+   :align: center
+
+When a user starts a new Kernel, the following steps process:
```

Comment:
```suggestion
When a user starts a new kernel, the following steps occur:
```

Reply:
I think "Kernel" should be "kernel" except when it's a proper noun (e.g., Mapping Kernel Manager, Kernel Spec Manager, etc.).  I believe this was also true in the general architecture section as well.  Perhaps a sweep can be made for such instances.

Reply:
Sounds good, I will change it to "kernel".